plugins/inventory-backend/src/service/standaloneApplication.ts

```diff
@@ -24,26 +24,29 @@ import cors from 'cors';
 import express from 'express';
 import helmet from 'helmet';
 import { Logger } from 'winston';
+import { Inventory } from '../inventory';
 import { createRouter } from './router';
 
 export interface ApplicationOptions {
   enableCors: boolean;
+  inventory: Inventory;
   logger: Logger;
 }
 
-export async function createApplication(
+export async function createStandaloneApplication(
```

Comment:
Shouldn't this be merged with `createServer`? I imagine we will want to ship a standard set of middleware with `backend-common` anyway.

Reply:
I am not sure. This left it perhaps unnecessarily complex, but the idea was that if you wanted to have several of these together in a single app shell, you could. I.e., serving on different ports.

This is getting a bit esoteric, sure. It's also not entirely clear that `run.ts` should be separate from the server.